insights/specs.py

```diff
@@ -176,7 +176,37 @@ def docker_container_ids(broker):
 httpd_conf = sf.glob_file(["/etc/httpd/conf/httpd.conf", "/etc/httpd/conf.d/*.conf"], name="httpd_conf"),
 httpd_conf_sos = sf.glob_file(["/conf/httpd/conf/httpd.conf", "/conf/httpd/conf.d/*.conf"], name="httpd_conf_sos", context=HostArchiveContext)
 httpd_error_log = sf.simple_file("var/log/httpd/error_log", name="httpd_error_log")
-httpd_pid = sf.simple_command("/bin/ps aux | grep /usr/sbin/httpd | grep -v grep | head -1 | awk '{print $2}'", name="httpd_pid")
+
```

Comment:
Replace all this with the following:

`httpd_pid = sf.simple_command("/bin/pgrep --oldest httpd", name="httpd_pid")`

Reply:
`pgrep` looks better, I will modify it in next commit. Thank you @skontar .

Reply:
@skontar - I have updated and tested the related datasources. Please help check if they are OK. Thanks.